### PR TITLE
Translate Source Maps and Strict Checks to ko

### PR DESCRIPTION
**Pull request by bumkeyy**
en:
- [Source_Map_Options_6175.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//tsconfig-reference/copy/en/categories/Source_Map_Options_6175.md)
- [Strict_Type_Checking_Options_6173.md](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//tsconfig-reference/copy/en/categories/Strict_Type_Checking_Options_6173.md)

번역 리뷰 요청합니다 : @yeonjuan @guyeol @dvlprsh

ref: #910

### CODE REVIEW

File: packages/tsconfig-reference/copy/ko/categories/Source_Map_Options_6175.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+display: "Source Maps"
+---
+
+TypeScript는 JavaScript 소스 맵 표준에 적합한 추가적인 파일을 생성하여,
+개발자가 이해할 수 있도록 충돌 현상과 풍부한 디버깅 툴을 제공합니다.
```
Comment (guyeol) [inline packages/tsconfig-reference/copy/ko/categories/Source_Map_Options_6175.md]:
```suggestion
개발자가 이해할 수 있는 풍부한 디버깅 툴과 충돌 보고서를 제공합니다.
```
`crash reports`은 `충돌 보고서`가 맞는 것 같습니다

File: packages/tsconfig-reference/copy/ko/categories/Strict_Type_Checking_Options_6173.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+TypeScript는 광범위한 JavaScript 패턴과 기본 값을 지원해 이런 스타일을 수용하는 상당한 유연성을 가집니다.
+종종 코드베이스의 안전성과 잠재적 확장성은 이런 기술 중 일부와 상충 될 수 있습니다.
+
+다양한 JavaScript 버전을 지원하기 때문에, TypeScript를 새 버전으로 업데이트하면 두 가지 유형의 오류를 발견할 수 있습니다.
+
+- TypeScript가 JavaScript의 이해도를 개선해 발견한 코드베이스에 이미 존재했던 오류들.
+- 새로운 문제 영역을 다루는 새로운 오류 모음.
+
+일반적으로 TypeScript는 후자의 오류 집합에 대한 컴파일 플래그를 추가하며, 기본적으로 활성화하지 않습니다.
```
Comment (guyeol) [inline packages/tsconfig-reference/copy/ko/categories/Strict_Type_Checking_Options_6173.md]:
```suggestion
일반적으로 TypeScript는 후자의 오류 집합에 대한 컴파일러 플래그를 추가하며, 기본적으로 비활성화됩니다.
```
`compile flag`와 `compiler flag`는 좀 다른 의미 같습니다

**Comment by bumkeyy** [comment]
@guyeol 
수정 완료했습니다 👍

**Comment by bumkeyy** [comment]
LGTM

**Pull request merged**